compose.override.yaml.dist

```
@@ -0,0 +1,4 @@
+services:
+  php:
+    volumes:
+      - ~/.composer/:/.cache/composer/
```

It's not that simple, as it depends on whether your (the host) environment follows the XDG specification or not.

If it doesn't, we should use:
```suggestion
    environment:
      COMPOSER_HOME: /tmp
    volumes:
      - ${COMPOSER_HOME:-$HOME/.composer}:/tmp
```

But if it does (like mine), it would be this instead:
```suggestion
    environment:
      - COMPOSER_HOME
      - COMPOSER_CACHE_DIR
    volumes:
      - ${COMPOSER_HOME:-$HOME/.config/composer}:$COMPOSER_HOME
      - ${COMPOSER_CACHE_DIR:-$HOME/.cache/composer}:$COMPOSER_CACHE_DIR
```